src/audio/sample_format.h

```diff
@@ -27,18 +32,20 @@
  * REMEMBER: changes here will require changes in lookup tables in both
  * sample_formats.cpp and audio_sink.cpp.
  */
-enum class SampleFormat : std::uint8_t {
-	PACKED_UNSIGNED_INT_8, ///< Packed 8-bit unsigned integer.
-	PACKED_SIGNED_INT_8,   ///< Packed 8-bit signed integer.
-	PACKED_SIGNED_INT_16,  ///< Packed 16-bit signed integer.
-	PACKED_SIGNED_INT_32,  ///< Packed 32-bit signed integer.
-	PACKED_FLOAT_32        ///< Packed 32-bit floating point.
+enum class Sample_format : std::uint8_t {
+	uint8,  ///< Packed 8-bit unsigned integer.
+	sint8,  ///< Packed 8-bit signed integer.
+	sint16, ///< Packed 16-bit signed integer.
+	sint32, ///< Packed 32-bit signed integer.
+	float32 ///< Packed 32-bit floating point.
 };
 
-/// Number of sample formats available; should agree with SampleFormat.
-constexpr std::size_t SAMPLE_FORMAT_COUNT = 5;
+/// Number of sample formats available; should agree with Sample_format.
+constexpr std::size_t sample_format_count = 5;
```

Comment:
constants should definitely be ALL_CAPS, don't understand this change

Reply:
It's [this ruley boi](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rl-all-caps)

Reply:
I can still disagree with it though :)
![](http://i.imgur.com/WT6Hk.gif)

Reply:
True, it is a bit of a weird one, and we don't really use macros enough for it to make much sense.